tutorcelery/patches/openedx-common-settings

```diff
@@ -0,0 +1 @@
+EXPLICIT_QUEUES.update({{CELERY_EXPLICIT_QUEUES}})
```

Comment:
Rendering the `CELERY_EXPLICIT_QUEUES` variable this way can lead to errors. We need to render this in a format Python can understand. One option could be:

```suggestion
EXPLICIT_QUEUES.update(json.loads('{{CELERY_EXPLICIT_QUEUES | tojson}}'))
```
but importing `json` is annoying. Other option could be the iteration over the object to assign every key to the corresponding value

Reply:
The other option is to remove this feature so `EXPLICIT_QUEUES` is not modified

Reply:
When using python files, this gets rendered correctly, we can tweak the patch to render it correctly

Reply:
From what we saw, it renders OK